python/sidecar/http.py

```diff
@@ -230,10 +224,10 @@ async def delete_project(project_id: str):
 
 
 @project_api.get("/{project_id}/files")
-async def get_project_files(project_id: str):
+async def get_project_files(project_id: str) -> ProjectFileTreeResponse:
```

Comment:
This return type is ok, but only with the project files and not the id, name, ...